readme.md

```diff
@@ -20,8 +20,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://MariCodec.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://MariCodec.github.io/layout_product-cards/report/html_report/)
 
```

Comment:
Have you run deploy because your demo and test links don't work?
![image](https://github.com/mate-academy/layout_product-cards/assets/46566190/0e2eec44-c262-479d-820d-c2e8377a0af1)